.eslintrc.json

```diff
@@ -29,7 +29,7 @@
       "import/no-unresolved": [2, { "commonjs": true }],
       "no-shadow": ["error", { "allow": ["req", "res", "err"] }],
       "valid-jsdoc": [
-        "error",
+        "off",
```

Comment:
why did you remove this it's needed and jsondoc where needed or ingore in file but not entire projects

Reply:
Resolved